dtfs-central-api/api-tests/v1/durable-functions/durable-functions.api-test.js

```diff
@@ -56,7 +56,7 @@ describe('DELETE /v1/portal/durable-functions', () => {
       })
       .to(`/v1/portal/durable-functions`);
 
-    expect(firstStatus).toBe(200);
-    expect(secondStatus).toBe(200);
+    expect(firstStatus).toEqual(200);
```

Comment:
should we change these to HttpStatusCode.Ok?

Reply:
That'd be good but I think should be done separately as there are ~400 instances that could be replaced.